src/modules/roc_core/target_posix/roc_core/fast_random.h

```diff
@@ -6,23 +6,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-//! @file roc_core/target_posix/roc_core/random.h
+//! @file roc_core/target_posix/roc_core/fast_random.h
 //! @brief Helpers to work with random numbers.
 
-#ifndef ROC_CORE_RANDOM_H_
-#define ROC_CORE_RANDOM_H_
+#ifndef ROC_CORE_FAST_RANDOM_H_
+#define ROC_CORE_FAST_RANDOM_H_
 
 #include "roc_core/stddefs.h"
 
 namespace roc {
 namespace core {
 
-//! Get random integer.
+//! Get non-choreographically-strong but faster random integer.
```

Comment:
"choreographically" was a typo, obviously. Please don't blindly copy-paste what I'm writing :)

Also, "cryptographically strong" is the property of "(pseudo) random integer *generator*" (PRNG), not the random integer itself. So it should be something:

> Get a random integer from a non-cryptographically strong, but fast PRNG.

Also, it seems a more widely used term is "cryptographically secure" instead of "cryptographically strong".